$OpenBSD: patch-configure,v 1.1 2009/11/19 15:35:53 jasper Exp $

Disable xulrunner 1.9, which will be picked up before 1.8, if's installed.

--- configure.orig	Wed Sep  3 20:44:43 2008
+++ configure	Thu Nov 19 16:33:18 2009
@@ -24495,7 +24495,8 @@ fi
 
 
 if test "x$enable_xulrunner" = "xyes" ; then
-
+# Force disable xulrunner 1.9, which will be picked up when installed
+if false; then
    { echo "$as_me:$LINENO: checking for XulRunner 1.9+ support" >&5
 echo $ECHO_N "checking for XulRunner 1.9+ support... $ECHO_C" >&6; }
 
@@ -24569,7 +24570,7 @@ else
 echo "${ECHO_T}yes" >&6; }
 	XULRUNNER_PROVIDER=libxul-embedding
 fi
-
+fi	# Force disable 1.9
    if test "x$XULRUNNER_PROVIDER" = "x" ; then
       { echo "$as_me:$LINENO: checking for XulRunner 1.8 support" >&5
 echo $ECHO_N "checking for XulRunner 1.8 support... $ECHO_C" >&6; }
